docs/modules/ROOT/pages/references/config-reference.adoc

```diff
@@ -0,0 +1,43 @@
+= Configuration Reference
+
+The Operator has two ways for configuration:
+
+. Per namespace backups. Optimal for shared clusters.
+. Global settings with namespaced schedules. Optimal for private clusters.
+
+== Environment Variables
+
+`BACKUP_ANNOTATION`:: the annotation to be used for filtering, default: `k8up.syn.tools/backup`
+`BACKUP_BACKUPCOMMANDANNOTATION`:: set the annotation name where the backup commands are stored, default `k8up.syn.tools/backupcommand`
+`BACKUP_CHECKSCHEDULE`:: the default check schedule, default: `0 0 * * 0`
+`BACKUP_DATAPATH`:: where the PVCs should get mounted in the container, default `/data`
+`BACKUP_FILEEXTENSIONANNOTATION`:: set the annotation name where the file extension is stored for backupcommands, default `k8up.syn.tools/file-extension`
+`BACKUP_GLOBALACCESSKEYID`:: set the S3 access key id to be used globaly
+`BACKUP_GLOBALKEEPJOBS`:: set the count of jobs to keep globally
+`BACKUP_GLOBALREPOPASSWORD`:: set the restic repository password to be used globaly
+`BACKUP_GLOBALRESTORES3ACCESKEYID`:: set the global restore S3 accessKeyID for restores
```

Comment:
Do these perhaps default to the non-restore S3 pendant, i.e. `BACKUP_GLOBALACCESSKEYID` in this case?

Reply:
I have no idea :shrug: